recipes/r-acepack/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = '1.4.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-acepack
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: acepack_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/acepack_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/acepack/acepack_{{ version }}.tar.gz
+
+  sha256: 82750507926f02a696f6cc03693e8d4a5ee7e92500c8c15a16a9c12addcd28b9
+
+build:
+  skip: true  # [win32]
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
```

Comment:
Will these play nice with conda-smithy without the `# [not win32]`? I lost track of that.